src/components/PointerCapture/__tests__/PointerCapture-test.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PointerCapture, { Props, Status } from '../PointerCapture';
+
+describe('PointerCapture', () => {
+    const defaults: Props = {
+        onDrawStart: jest.fn(),
+        onDrawStop: jest.fn(),
+        onDrawUpdate: jest.fn(),
+        onMouseOut: jest.fn(),
+        onMouseOver: jest.fn(),
+        status: Status.init,
+    };
+
+    type CallbackTypes = keyof Props;
+
+    const getWrapper = (props = {}): ShallowWrapper => shallow(<PointerCapture {...defaults} {...props} />);
+
+    beforeEach(() => {
+        jest.spyOn(React, 'useEffect').mockImplementation(noop);
+    });
+
+    describe('render', () => {
+        test('should render with default props', () => {
+            const wrapper = getWrapper();
+
+            expect(wrapper.find('div').props()).toMatchObject({
+                onClick: expect.any(Function),
+                onMouseDown: expect.any(Function),
+                onMouseOut: expect.any(Function),
+                onMouseOver: expect.any(Function),
+                onTouchCancel: expect.any(Function),
+                onTouchEnd: expect.any(Function),
+                onTouchMove: expect.any(Function),
+                onTouchStart: expect.any(Function),
+                role: 'presentation',
+            });
+        });
+
+        test('should render with provided props', () => {
+            const wrapper = getWrapper({ children: <span className="child" />, className: 'foo', 'data-bar': 'bar' });
+
+            expect(wrapper.find('div').props()).toMatchObject({
+                className: 'foo',
+                'data-bar': 'bar',
+                onClick: expect.any(Function),
+                onMouseDown: expect.any(Function),
+                onMouseOut: expect.any(Function),
+                onMouseOver: expect.any(Function),
+                onTouchCancel: expect.any(Function),
+                onTouchEnd: expect.any(Function),
+                onTouchMove: expect.any(Function),
+                onTouchStart: expect.any(Function),
+                role: 'presentation',
+            });
+            expect(wrapper.find('span.child').exists()).toBe(true);
+        });
+    });
+
+    describe('eventing', () => {
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
We should try to avoid disabling the type system when possible. I don't think this was done in the Region Creator tests?